packages/reactotron-server/src/app/screens/Timeline/timeline-screen.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { Query } from "react-apollo"
+import gql from "graphql-tag"
+
+import { TimelineCommandsList } from "./timeline-commands-list"
+
+const COMMAND_QUERY = gql`
+  {
+    commands {
+      messageId
+      type
+      date
+      clientId
+      payload
+    }
+  }
+`
+const COMMAND_SUB = gql`
+  subscription {
+    commandAdded {
+      messageId
+      type
+      date
+      clientId
+      payload
+    }
+  }
+`
+
+export class TimelineScreen extends React.Component {
+  render() {
+    return (
+      <div>
+        If everything works right you should see a lot of commands here:{" "}
+        <div>
+          <Query query={COMMAND_QUERY}>
```

Comment:
This query takes a little bit when switching to the timeline page if there are a lot of events. Might need to look at only loading the last 10-20 or so and then allow for infinite scrolling or something of that nature.